File: algorithms/interview-preparation-kit-1/js/code/repeatedString/repeatedString.print.mjs
import { repeatedString } from './repeatedString.mjs';

console.log(repeatedString('a', 10));
console.log(10);

console.log(repeatedString('b', 10));
console.log(0);

console.log(repeatedString('aba', 10));
console.log(7);

console.log(repeatedString('a', 1000000000000));
console.log(1000000000000);

File: algorithms/interview-preparation-kit-1/ruby/code-response/repeatedString/repeated_string.print.rb
require_relative "./repeated_string"

puts(RepeatedString.repeat('a', 10))
puts(10)

puts(RepeatedString.repeat('b', 10))
puts(0)

puts(RepeatedString.repeat('aba', 10))
puts(7)

puts(RepeatedString.repeat('a', 1000000000000))
puts(1000000000000)